src/lib/kebab-case-to-camel-case.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function kebabCaseToCamelCase(kebab) {
```

Comment:
We didn't have it before. The utility function included in `src/views/payment-sheet-views/card-view.js` goes the other way, it converts from camelcase to kebab case.

Unless you mean something else?